# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/iptraf-ng/hotfix-stdbool.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- iptraf-ng-1.2.1/src/iptraf-ng-compat.h.vanilla	2025-05-04 17:24:23.837100498 +0200
+++ iptraf-ng-1.2.1/src/iptraf-ng-compat.h	2025-05-04 17:24:37.570918009 +0200
@@ -19,6 +19,7 @@
 #include <poll.h>
 #include <limits.h>
 #include <locale.h>
+#include <stdbool.h>
 
 #include <sys/types.h>
 #include <sys/stat.h>
